### ISSUE TITLE
Strange train log of Kinetics

### ISSUE THREAD

**Issue by zhangpur** [yysijie/st-gcn#46]
I try to train Kinetics-skeleton by myself, but get confused with the training log. 
Here is the output log of my training process, where the loss seems hard to converge, and the first validation accuracy is only 2.07%. (2 GPU used & batch size=128)
Is it normal or not?Thanks for your help!

[ Wed Jun  6 16:01:01 2018 ] Training epoch: 1
[ Wed Jun  6 16:01:16 2018 ] 	Batch(0/1879) done. Loss: 6.3596  lr:0.100000
[ Wed Jun  6 16:02:39 2018 ] 	Batch(100/1879) done. Loss: 5.9635  lr:0.100000
[ Wed Jun  6 16:04:06 2018 ] 	Batch(200/1879) done. Loss: 5.8815  lr:0.100000
[ Wed Jun  6 16:05:30 2018 ] 	Batch(300/1879) done. Loss: 5.8872  lr:0.100000
[ Wed Jun  6 16:06:55 2018 ] 	Batch(400/1879) done. Loss: 5.9088  lr:0.100000
[ Wed Jun  6 16:08:20 2018 ] 	Batch(500/1879) done. Loss: 5.8656  lr:0.100000
[ Wed Jun  6 16:09:44 2018 ] 	Batch(600/1879) done. Loss: 5.8867  lr:0.100000
[ Wed Jun  6 16:11:09 2018 ] 	Batch(700/1879) done. Loss: 5.8330  lr:0.100000
[ Wed Jun  6 16:12:33 2018 ] 	Batch(800/1879) done. Loss: 5.9027  lr:0.100000
[ Wed Jun  6 16:14:00 2018 ] 	Batch(900/1879) done. Loss: 5.8755  lr:0.100000
[ Wed Jun  6 16:15:25 2018 ] 	Batch(1000/1879) done. Loss: 5.8760  lr:0.100000
[ Wed Jun  6 16:16:50 2018 ] 	Batch(1100/1879) done. Loss: 5.8521  lr:0.100000
[ Wed Jun  6 16:18:14 2018 ] 	Batch(1200/1879) done. Loss: 5.8933  lr:0.100000
[ Wed Jun  6 16:19:38 2018 ] 	Batch(1300/1879) done. Loss: 5.9311  lr:0.100000
[ Wed Jun  6 16:21:03 2018 ] 	Batch(1400/1879) done. Loss: 5.8277  lr:0.100000
[ Wed Jun  6 16:22:28 2018 ] 	Batch(1500/1879) done. Loss: 5.8691  lr:0.100000
[ Wed Jun  6 16:23:53 2018 ] 	Batch(1600/1879) done. Loss: 5.8348  lr:0.100000
[ Wed Jun  6 16:25:18 2018 ] 	Batch(1700/1879) done. Loss: 5.9588  lr:0.100000
[ Wed Jun  6 16:26:43 2018 ] 	Batch(1800/1879) done. Loss: 5.9448  lr:0.100000
[ Wed Jun  6 16:27:48 2018 ] 	Mean training loss: 5.9060.
[ Wed Jun  6 16:27:48 2018 ] 	Time consumption: [Data]01%, [Network]99%
[ Wed Jun  6 16:27:48 2018 ] Training epoch: 2
[ Wed Jun  6 16:28:01 2018 ] 	Batch(0/1879) done. Loss: 5.8867  lr:0.100000
[ Wed Jun  6 16:29:25 2018 ] 	Batch(100/1879) done. Loss: 5.9181  lr:0.100000
[ Wed Jun  6 16:30:49 2018 ] 	Batch(200/1879) done. Loss: 5.8454  lr:0.100000
[ Wed Jun  6 16:32:13 2018 ] 	Batch(300/1879) done. Loss: 5.8444  lr:0.100000
[ Wed Jun  6 16:33:36 2018 ] 	Batch(400/1879) done. Loss: 5.9084  lr:0.100000
[ Wed Jun  6 16:34:59 2018 ] 	Batch(500/1879) done. Loss: 5.8634  lr:0.100000
[ Wed Jun  6 16:36:22 2018 ] 	Batch(600/1879) done. Loss: 5.8313  lr:0.100000
[ Wed Jun  6 16:37:45 2018 ] 	Batch(700/1879) done. Loss: 5.9054  lr:0.100000
[ Wed Jun  6 16:39:09 2018 ] 	Batch(800/1879) done. Loss: 5.8981  lr:0.100000
[ Wed Jun  6 16:40:32 2018 ] 	Batch(900/1879) done. Loss: 5.9091  lr:0.100000
[ Wed Jun  6 16:41:56 2018 ] 	Batch(1000/1879) done. Loss: 5.8667  lr:0.100000
[ Wed Jun  6 16:43:19 2018 ] 	Batch(1100/1879) done. Loss: 5.9404  lr:0.100000
[ Wed Jun  6 16:44:42 2018 ] 	Batch(1200/1879) done. Loss: 5.8945  lr:0.100000
[ Wed Jun  6 16:46:05 2018 ] 	Batch(1300/1879) done. Loss: 5.9334  lr:0.100000
[ Wed Jun  6 16:47:28 2018 ] 	Batch(1400/1879) done. Loss: 5.8050  lr:0.100000
[ Wed Jun  6 16:48:50 2018 ] 	Batch(1500/1879) done. Loss: 5.9695  lr:0.100000
[ Wed Jun  6 16:50:13 2018 ] 	Batch(1600/1879) done. Loss: 5.8498  lr:0.100000
[ Wed Jun  6 16:51:36 2018 ] 	Batch(1700/1879) done. Loss: 5.8443  lr:0.100000
[ Wed Jun  6 16:52:59 2018 ] 	Batch(1800/1879) done. Loss: 5.8773  lr:0.100000
[ Wed Jun  6 16:54:03 2018 ] 	Mean training loss: 5.8806.
[ Wed Jun  6 16:54:03 2018 ] 	Time consumption: [Data]01%, [Network]99%
[ Wed Jun  6 16:54:03 2018 ] Training epoch: 3
[ Wed Jun  6 16:54:15 2018 ] 	Batch(0/1879) done. Loss: 5.8311  lr:0.100000
[ Wed Jun  6 16:55:38 2018 ] 	Batch(100/1879) done. Loss: 5.8930  lr:0.100000
[ Wed Jun  6 16:57:00 2018 ] 	Batch(200/1879) done. Loss: 5.8610  lr:0.100000
[ Wed Jun  6 16:58:24 2018 ] 	Batch(300/1879) done. Loss: 5.8864  lr:0.100000
[ Wed Jun  6 16:59:47 2018 ] 	Batch(400/1879) done. Loss: 5.8617  lr:0.100000
[ Wed Jun  6 17:01:10 2018 ] 	Batch(500/1879) done. Loss: 5.8742  lr:0.100000
[ Wed Jun  6 17:02:33 2018 ] 	Batch(600/1879) done. Loss: 5.8479  lr:0.100000
[ Wed Jun  6 17:03:56 2018 ] 	Batch(700/1879) done. Loss: 5.8783  lr:0.100000
[ Wed Jun  6 17:05:18 2018 ] 	Batch(800/1879) done. Loss: 5.8687  lr:0.100000
[ Wed Jun  6 17:06:41 2018 ] 	Batch(900/1879) done. Loss: 5.8906  lr:0.100000
[ Wed Jun  6 17:08:04 2018 ] 	Batch(1000/1879) done. Loss: 5.7582  lr:0.100000
[ Wed Jun  6 17:09:26 2018 ] 	Batch(1100/1879) done. Loss: 5.8391  lr:0.100000
[ Wed Jun  6 17:10:50 2018 ] 	Batch(1200/1879) done. Loss: 5.8959  lr:0.100000
[ Wed Jun  6 17:12:13 2018 ] 	Batch(1300/1879) done. Loss: 5.8328  lr:0.100000
[ Wed Jun  6 17:13:37 2018 ] 	Batch(1400/1879) done. Loss: 5.8172  lr:0.100000
[ Wed Jun  6 17:15:01 2018 ] 	Batch(1500/1879) done. Loss: 5.9253  lr:0.100000
[ Wed Jun  6 17:16:22 2018 ] 	Batch(1600/1879) done. Loss: 5.8176  lr:0.100000
[ Wed Jun  6 17:17:45 2018 ] 	Batch(1700/1879) done. Loss: 5.8348  lr:0.100000
[ Wed Jun  6 17:19:08 2018 ] 	Batch(1800/1879) done. Loss: 5.8490  lr:0.100000
[ Wed Jun  6 17:20:12 2018 ] 	Mean training loss: 5.8631.
[ Wed Jun  6 17:20:12 2018 ] 	Time consumption: [Data]01%, [Network]99%
[ Wed Jun  6 17:20:12 2018 ] Training epoch: 4
[ Wed Jun  6 17:20:24 2018 ] 	Batch(0/1879) done. Loss: 5.8593  lr:0.100000
[ Wed Jun  6 17:21:46 2018 ] 	Batch(100/1879) done. Loss: 5.8140  lr:0.100000
[ Wed Jun  6 17:23:09 2018 ] 	Batch(200/1879) done. Loss: 5.8602  lr:0.100000
[ Wed Jun  6 17:24:32 2018 ] 	Batch(300/1879) done. Loss: 5.7933  lr:0.100000
[ Wed Jun  6 17:25:55 2018 ] 	Batch(400/1879) done. Loss: 5.8226  lr:0.100000
[ Wed Jun  6 17:27:18 2018 ] 	Batch(500/1879) done. Loss: 5.8541  lr:0.100000
[ Wed Jun  6 17:28:41 2018 ] 	Batch(600/1879) done. Loss: 5.8988  lr:0.100000
[ Wed Jun  6 17:30:03 2018 ] 	Batch(700/1879) done. Loss: 5.8665  lr:0.100000
[ Wed Jun  6 17:31:26 2018 ] 	Batch(800/1879) done. Loss: 5.8884  lr:0.100000
[ Wed Jun  6 17:32:49 2018 ] 	Batch(900/1879) done. Loss: 5.8810  lr:0.100000
[ Wed Jun  6 17:34:12 2018 ] 	Batch(1000/1879) done. Loss: 5.9417  lr:0.100000
[ Wed Jun  6 17:35:35 2018 ] 	Batch(1100/1879) done. Loss: 5.8325  lr:0.100000
[ Wed Jun  6 17:36:58 2018 ] 	Batch(1200/1879) done. Loss: 5.8373  lr:0.100000
[ Wed Jun  6 17:38:21 2018 ] 	Batch(1300/1879) done. Loss: 5.9107  lr:0.100000
[ Wed Jun  6 17:39:43 2018 ] 	Batch(1400/1879) done. Loss: 5.8672  lr:0.100000
[ Wed Jun  6 17:41:07 2018 ] 	Batch(1500/1879) done. Loss: 5.8756  lr:0.100000
[ Wed Jun  6 17:42:31 2018 ] 	Batch(1600/1879) done. Loss: 5.8735  lr:0.100000
[ Wed Jun  6 17:43:54 2018 ] 	Batch(1700/1879) done. Loss: 5.8137  lr:0.100000
[ Wed Jun  6 17:45:16 2018 ] 	Batch(1800/1879) done. Loss: 5.8421  lr:0.100000
[ Wed Jun  6 17:46:20 2018 ] 	Mean training loss: 5.8499.
[ Wed Jun  6 17:46:20 2018 ] 	Time consumption: [Data]01%, [Network]99%
[ Wed Jun  6 17:46:20 2018 ] Training epoch: 5
[ Wed Jun  6 17:46:32 2018 ] 	Batch(0/1879) done. Loss: 5.8172  lr:0.100000
[ Wed Jun  6 17:47:54 2018 ] 	Batch(100/1879) done. Loss: 5.7880  lr:0.100000
[ Wed Jun  6 17:49:18 2018 ] 	Batch(200/1879) done. Loss: 5.8781  lr:0.100000
[ Wed Jun  6 17:50:40 2018 ] 	Batch(300/1879) done. Loss: 5.8889  lr:0.100000
[ Wed Jun  6 17:52:02 2018 ] 	Batch(400/1879) done. Loss: 5.8591  lr:0.100000
[ Wed Jun  6 17:53:26 2018 ] 	Batch(500/1879) done. Loss: 5.8163  lr:0.100000
[ Wed Jun  6 17:54:49 2018 ] 	Batch(600/1879) done. Loss: 5.7984  lr:0.100000
[ Wed Jun  6 17:56:12 2018 ] 	Batch(700/1879) done. Loss: 5.8237  lr:0.100000
[ Wed Jun  6 17:57:35 2018 ] 	Batch(800/1879) done. Loss: 5.7660  lr:0.100000
[ Wed Jun  6 17:58:58 2018 ] 	Batch(900/1879) done. Loss: 5.7854  lr:0.100000
[ Wed Jun  6 18:00:21 2018 ] 	Batch(1000/1879) done. Loss: 5.8939  lr:0.100000
[ Wed Jun  6 18:01:44 2018 ] 	Batch(1100/1879) done. Loss: 5.8935  lr:0.100000
[ Wed Jun  6 18:03:06 2018 ] 	Batch(1200/1879) done. Loss: 5.7438  lr:0.100000
[ Wed Jun  6 18:04:29 2018 ] 	Batch(1300/1879) done. Loss: 5.7744  lr:0.100000
[ Wed Jun  6 18:05:52 2018 ] 	Batch(1400/1879) done. Loss: 5.8634  lr:0.100000
[ Wed Jun  6 18:07:15 2018 ] 	Batch(1500/1879) done. Loss: 5.9241  lr:0.100000
[ Wed Jun  6 18:08:38 2018 ] 	Batch(1600/1879) done. Loss: 5.8004  lr:0.100000
[ Wed Jun  6 18:10:00 2018 ] 	Batch(1700/1879) done. Loss: 5.8693  lr:0.100000
[ Wed Jun  6 18:11:22 2018 ] 	Batch(1800/1879) done. Loss: 5.7603  lr:0.100000
[ Wed Jun  6 18:12:27 2018 ] 	Mean training loss: 5.8416.
[ Wed Jun  6 18:12:27 2018 ] 	Time consumption: [Data]01%, [Network]99%
[ Wed Jun  6 18:12:27 2018 ] Eval epoch: 5
[ Wed Jun  6 18:13:59 2018 ] 	Mean test loss of 155 batches: 5.710531379330543.
[ Wed Jun  6 18:14:00 2018 ] 	Top1: 2.07%
[ Wed Jun  6 18:14:00 2018 ] 	Top5: 8.24%

**Comment by zhangpur** [comment]
No, I use the version 0.1.0. This must be the point.
Things goes okay when I try NTU dataset with the old version code.

Thank you for reply!
